ocaml/idl/datamodel_pool.ml

```diff
@@ -712,7 +712,7 @@ open Datamodel_types
         Ref _pool, "self", "The pool";
         Set (Ref _repository), "value", "The set of repositories to be enabled"
       ]
-      ~allowed_roles:_R_POOL_ADMIN
+      ~allowed_roles:_R_POOL_OP
```

Comment:
Do we have a baked-in gpg key and we only trust RPMs signed by that gpg key when using the repository?
(I think this is what happens with updates, but I'm not sure what happens with a Yum repository, do we just trust all RPMs inside it by default, or only those signed by us?) 
If not then pool operator could escalate itself to pool admin by creating a Yum repository itself, replacing XAPI rpm with one that disables all authentication.

Reply:
Yes indeed, that is the reason: you can only install signed RPMs, and a pool-operator cannot change which key is accepted.

Reply:
I assume we (will) have a test for that.